[e2e] Global operator is crashing

```
{"level":"info","@timestamp":"2019-11-21T13:09:46.612Z","logger":"manager","message":"Setting up client for manager","ver":"1.0.0-beta1-e56f1fec"}
{"level":"info","@timestamp":"2019-11-21T13:09:46.612Z","logger":"manager","message":"Setting up scheme","ver":"1.0.0-beta1-e56f1fec"}
{"level":"info","@timestamp":"2019-11-21T13:09:46.613Z","logger":"manager","message":"Setting up manager","ver":"1.0.0-beta1-e56f1fec"}
{"level":"info","@timestamp":"2019-11-21T13:09:46.613Z","logger":"manager","message":"Operator configured to manage all namespaces","ver":"1.0.0-beta1-e56f1fec"}
{"level":"info","@timestamp":"2019-11-21T13:09:47.118Z","logger":"controller-runtime.metrics","message":"metrics server is starting to listen","ver":"1.0.0-beta1-e56f1fec","addr":":0"}
{"level":"info","@timestamp":"2019-11-21T13:09:47.127Z","logger":"manager","message":"Setting up controllers","ver":"1.0.0-beta1-e56f1fec","roles":["global","webhook"]}
{"level":"info","@timestamp":"2019-11-21T13:09:47.127Z","logger":"manager","message":"Automatic management of the webhook certificates enabled","ver":"1.0.0-beta1-e56f1fec"}
{"level":"error","@timestamp":"2019-11-21T13:09:47.136Z","logger":"manager","message":"unable to setup and fill the webhook certificates","ver":"1.0.0-beta1-e56f1fec","error":"validatingwebhookconfigurations.admissionregistration.k8s.io \"elastic-webhook.k8s.elastic.co\" not found","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128\ngithub.com/elastic/cloud-on-k8s/cmd/manager.setupWebhook\n\t/go/src/github.com/elastic/cloud-on-k8s/cmd/manager/main.go:351\ngithub.com/elastic/cloud-on-k8s/cmd/manager.execute\n\t/go/src/github.com/elastic/cloud-on-k8s/cmd/manager/main.go:281\ngithub.com/elastic/cloud-on-k8s/cmd/manager.glob..func1\n\t/go/src/github.com/elastic/cloud-on-k8s/cmd/manager/main.go:75\ngithub.com/spf13/cobra.(*Command).execute\n\t/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\t/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914\ngithub.com/spf13/cobra.(*Command).Execute\n\t/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864\nmain.main\n\t/go/src/github.com/elastic/cloud-on-k8s/cmd/main.go:27\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"}
```

This is because the webhook is not created during the e2e setup: 

```
 > k get ValidatingWebhookConfiguration
No resources found.
```

This is the reason of the failure of the following test:

```
14:17:15      --- FAIL: TestEnterpriseLicenseSingle/Elasticsearch_license_should_be_[gold_platinum] (300.00s)
14:17:15          utils.go:80: 
14:17:15              	Error Trace:	utils.go:80
14:17:15              	Error:      	Received unexpected error:
14:17:15              	            	expectedTypes license type [gold platinum] got basic
14:17:15              	Test:       	TestEnterpriseLicenseSingle/Elasticsearch_license_should_be_[gold_platinum]
```

--- a/config/e2e/global_operator.yaml
+++ b/config/e2e/global_operator.yaml
@@ -186,11 +186,11 @@ spec:
             cpu: 100m
             memory: 20Mi
         ports:
-        - containerPort: 9876
+        - containerPort: 9443
           name: webhook-server
           protocol: TCP
         volumeMounts:
-        - mountPath: /tmp/cert
+        - mountPath: /tmp/k8s-webhook-server/serving-certs
           name: cert
           readOnly: true
       terminationGracePeriodSeconds: 10
@@ -199,3 +199,41 @@ spec:
         secret:
           defaultMode: 420
           secretName: webhook-server-secret
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: elastic-webhook.k8s.elastic.co
+  namespace: {{ .GlobalOperator.Namespace }}
+webhooks:
+  - clientConfig:
+      caBundle: Cg==
+      service:
+        name: elastic-webhook-server
+        namespace: {{ .GlobalOperator.Namespace }}
+        # this is the path controller-runtime automatically generates
+        path: /validate-elasticsearch-k8s-elastic-co-v1beta1-elasticsearch
+    failurePolicy: Ignore
+    name: elastic-es-validation.k8s.elastic.co
+    rules:
+      - apiGroups:
+          - elasticsearch.k8s.elastic.co
+        apiVersions:
+          - v1beta1
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - elasticsearches
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: elastic-webhook-server
+  namespace: {{ .GlobalOperator.Namespace }}
+spec:
+  ports:
+    - port: 443
+      targetPort: 9443
+  selector:
+    control-plane: elastic-operator

